src/clients/ntp-client_proposal.rb

```diff
@@ -89,56 +89,46 @@ def main
     end
 
     def ui_help_text
-      # help text
-      tmp = _(
-        "<p>Press <b>Synchronize Now</b>, to get your system time set correctly " \
-        "using the selected NTP server. If you want to make use of NTP permanently, " \
-        "enable the <b>Save NTP Configuration</b> option</p>"
-      )
-
-      tmp = Ops.add(
-        tmp,
-        _(
+      if Stage.initial
+        # help text
+        tmp = _(
+          "<p>Press <b>Synchronize Now</b>, to get your system time set correctly " \
+          "using the selected NTP server. If you want to make use of NTP permanently, " \
+          "enable the <b>Save NTP Configuration</b> option</p>"
+        ) + _(
           "<p>Enabling <b>Run NTP as daemon</b> option, the NTP service will be " \
           "started as daemon. Otherwise the system time will be synchronized periodically. " \
           "The default interval is 15 min. You can change it after installation " \
           "with the <b>yast2 ntp-client module</b>.</p>"
+        ) + _(
+          "<p>Synchronization with the NTP server can be done only when " \
+          "the network is configured.</p>"
         )
-      )
-
-      # help text, cont.
-      if !Stage.initial
-        tmp = Ops.add(
-          tmp,
-          _(
-            "<p>Using the <b>Configure</b> button, open the advanced NTP configuration.</p>"
-          )
+      else
+        # help text
+        tmp = _(
+          "<p>Using the <b>Configure</b> button, open the advanced NTP configuration.</p>"
         )
       end
 
-      # help text, cont.
-      tmp = Ops.add(
-        tmp,
-        _(
-          "<p>Synchronization with the NTP server can be done only when " \
-          "the network is configured.</p>"
-        )
-      )
       tmp
```

Comment:
and now when I see it, whole tmp can be removed as it is not needed and can be simple method like
```
def a
  if Stage.initial
    _() + _() + _()
  else
    _()
  end
end